rct229/ruletest_engine/ruletest_jsons/system_types/System 12_CAV_SZ_HW.json

```diff
@@ -95,13 +90,7 @@
                 },
                 {
                     "id": "Chiller Loop 1",
-                    "type": "COOLING",
-                    "child_loops": [
-                        {
-                            "id": "Secondary CHW Loop 1",
-                            "type": "COOLING"
-                        }
-                    ]
+                    "type": "COOLING"
                 }
```

Comment:
I'm unsure why the child loops are being removed. I wonder if the system_types excel file on onedrive is out of date? It appears different than what I have locally and could be the source of this. You'll need the new script I uploaded in this PR to generate the JSONs from the spreadsheet, I believe:
https://github.com/pnnl/ruleset-checking-tool/pull/835

Reply:
Juan, as we discussed with each other, the system type master Excel file was updated based on your latest version (which includes the child loops). Could you check this out one more time?

Also, the system type master Excel file on OneDrive was updated.